tests/network/sriov.go

```diff
@@ -787,3 +784,15 @@ func sriovNodeName(sriovResourceName string) (string, error) {
 	}
 	return sriovNodes[0].Name, nil
 }
+
+func mountDeviceToFS(vmi *v1.VirtualMachineInstance, devName string) error {
```

Comment:
```suggestion
func mountGuestDevice(vmi *v1.VirtualMachineInstance, devName string) error {
```
nit: mounting is ALWAYS "to filesystem".

It's so much better without the lambda.

Reply:
I agree. [DONE](https://github.com/kubevirt/kubevirt/compare/f9b5bb28e4135a70722fd5c64a12ca6981241e20..072925a21561823f770f9fac163b69b28847c83e)